monero-outreach-docs/en/storyboard/wallet_quickstart_en.md

```diff
@@ -0,0 +1,57 @@
+# Monero Wallet Quickstart
+*04/22/19*	
+Almutasim
```

Comment:
Add two trailing spaces at the end of the strings to create a newline, please.

Reply:
Done